index.js

```diff
@@ -7,13 +7,26 @@ const YAML = require('js-yaml')
 const TOML = { parse: require('toml-j0.4').parse, stringify: require('tomlify-j0.4') }
 const INI = require('ini')
 
+/**
+ * Removes '.' in file extensions
+ * @param  {string} type input file extension
+ * @return {string}      file extension without '.'
+ */
+const validType = (type) => {
+    if (type.charAt(0) === '.') {
+        type = type.substr(1)
+    }
+    return type
+}
+
```

Comment:
The comment says `Remove '.' in file extensions`, the name of the function is `validType` -- that's kind of confusing. I'm a big fan of naming functions in such a way that you immediately know what they are doing. So I think something like `removeFileExt` would be a good name.

Also the code can be simplified: 
```js
const removeFileExt = type =>
    type.startsWith('.') ? type.substr(1) : type
```